stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -328,8 +328,7 @@ BitMask BuiltinTypeInfo::getSpareBits(TypeConverter &TC, bool &hasAddrOnly) cons
     mask.keepOnlyMostSignificantBits(getSize() * 8 - intSize);
     return mask;
   } else if (
-    Name == "ypXp" || // Any.Type
-    Name == "yyXf" // 'yyXf' =  @thin () -> Void function
+    Name == "ypXp" // Any.Type
```

Comment:
Could you run clang-format on these changes?

Reply:
I'll see how much damage that does.

Reply:
Hmmm....  that adds about 340 lines of whitespace changes.  I'll leave that to someone else.  I did adjust this line and one other line that's actually in this PR to better match clang-format style.

Reply:
You can run something like this command to only reformat your changes:

```
git diff -U0 HEAD^ | /path/to/llvm-project/clang/tools/clang-format/clang-format-diff.py -binary /path/to/swift/directory/build/Ninja-RelWithDebInfoAssert+stdlib-DebugAssert/llvm-macosx-arm64/bin/clang-format -i -p1
```